### DISCUSSION TITLE
```uno-check``` ```preview-major``` Fix failed - Workload Install failed

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#13891] [General]
![image](https://github.com/unoplatform/uno/assets/69314237/9ca7b04f-2ec9-45a9-a0d7-ba956eaa5521)

![image](https://github.com/unoplatform/uno/assets/69314237/b617f3e9-b8e4-4da6-950e-1e4fc1f0a292)

______________________________________

uno-check version: v1.15.0.0
executed command: ```uno-check --preview-major```
dotnet list skds:
```
6.0.317 [C:\Program Files\dotnet\sdk]
7.0.306 [C:\Program Files\dotnet\sdk]
7.0.400 [C:\Program Files\dotnet\sdk]
8.0.100-rc.1.23455.8 [C:\Program Files\dotnet\sdk]
8.0.100-rc.1.23463.5 [C:\Program Files\dotnet\sdk]
```

**Comment by limeniye** [comment]
How I fixed that.
I've just installed maui
```
dotnet workload install maui
```

But I don't see this point on my Visual Studio 2022 Preview 17.8.0 Installer:
![image](https://github.com/unoplatform/uno/assets/69314237/1da54e77-4fa1-404a-a124-7ed1eab11d42)

But I know that it's should be like this:
![vs17 7-p2-installer](https://github.com/unoplatform/uno/assets/69314237/886b336b-1e5c-4dd7-9694-0c656cc2ef50)

**Reply by assassin316** [reply]
Didn't work on my end

**Comment by assassin316** [comment]
Having same issue, and the Maui workload install didn't fix the issue.

**Reply by limeniye** [reply]
Please, show the ```dotnet --list-sdks```

**Reply by jeromelaban** [reply]
@assassin316 can you provide a log of your error? Make a new discussion, this discussion is related to an older behavior for net8 previews.

**Reply by assassin316** [reply]
Agnes help me with the issue. I had an outdated Uno-Check, after updating it everything worked fine 👍
